trikot-viewmodels-declarative-flow/compose-flow/src/main/kotlin/com/mirego/trikot/viewmodels/declarative/compose/extensions/ViewModelExtensions.kt

```diff
@@ -133,3 +135,12 @@ inline fun <T, S> S.asState(
     }
     return state
 }
+
+@Suppress("UNCHECKED_CAST")
+private  fun <T, V> KProperty<V>.callGetter(receiver: T): V =
```

Comment:
two spaces between private and fun

Reply:
ktlint seems not to run on those modules, I'll fix this.

Reply:
Ok so I setupped ktlint properly, lots of unrelated changes, sorry for that.